testing/it/http/src/test/java/com/adobe/cq/wcm/core/components/it/http/TeaserIT.java

```diff
@@ -0,0 +1,51 @@
+package com.adobe.cq.wcm.core.components.it.http;
```

Comment:
The build is failing because the file header is missing, reproducible locally by building the it subproject.